### DISCUSSION TITLE
Prisma Roadmap - Still actively maintained?

### DISCUSSION THREAD

**Discussion by psikoi** [prisma/prisma#13199] [Q&A] [answered]
[This page](https://www.prisma.io/docs/about/prisma/roadmap) on the docs directs us to the [Notion roadmap](https://www.notion.so/Prisma-Roadmap-50766227b779464ab98899accb98295f), but it seems it was last updated on the 7th of April.

Is this Notion file actively maintained? I'm a big Prisma fan and like to occasionally pop by that page to get a quick overview of what's to come, would be disappointing to see it forgotten.

**Comment by nurul3101** [comment, marked as answer]
Hey @psikoi 👋 

The notion document is actively maintained, it will be updated regularly!
The cadence isn't fixed but it's definitely the official source.

**Reply by psikoi** [reply]
Wonderful, thank you!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
